components/theme/Button.tsx

```diff
@@ -46,8 +45,9 @@ export const Button: React.FC<ButtonProps> = ({
   return (
     <button
       className={classes.join(' ')}
```

Comment:
What happens if someone passes in className as a prop? Will the passed-in className prop override line 47 ?`className={classes.join(' ')}`

Reply:
On line 28 can you try `const classes = ['btn', className]`
And if className doesnt exist(`aka not passed as a prop`) you can set default value of className as empty string

Reply:
> After line 28 can you try
> 
> ```
> className && classes.push(className)
> ```

Yeah this is a good thing to point out. It works, but I have to explicitly state the `className` prop

Reply:
https://github.com/mui/material-ui/blob/master/packages/mui-joy/src/Button/Button.tsx#L60

materialsui button uses styled components, but that currently isn't our setup for setting css in c0d3

Reply:
> > It works, but I have to explicitly pass in the `className` prop
> 
> I don't understand, what is the issue with doing that? Is there a better way to handle this use case for this repo?

Oh I have no issue with it, just more of an FYI since we were discussing trying to explicitly state as few props as possible.